### ISSUE TITLE
Can not find control

### ISSUE THREAD

**Issue by tungnguyendev** [icnocop/cuite#114]
Hi, 

I'm using CUITe fw for automation test, when I create 1 combobox and 1 numericUpDown control without AccessibleName(as the Screen Object sample in fw), I'm try to get control with By.ControlName but can not, whenever CUITe find the first control for click event. please help, thanks!
`public WinComboBox combobox
{
    get
    {
        return Find<WinComboBox>(By.ControlName("comboBox1"));
    }
}

public WinComboBox numericUpDown
{
    get
    {
        return Find<WinComboBox>(By.ControlName("numericUpDown1"));
    }
}`
And I call event
combobox.Click();
numericUpDown.Click();

**Comment by icnocop** [comment]
Hi.

Which version of CUITe are you using?
Have you tried upgrading to the latest version?

The Find method is a generic method and takes a type parameter but your code doesn't have one.

For example, I expected the following:
```
public WinComboBox combobox
{
    get
    {
        return Find<WinComboBox>(By.ControlName("comboBox1"));
    }
}

public WinEdit numericUpDown
{
    get
    {
        return Find<WinEdit>(By.ControlName("numericUpDown1"));
    }
}
```

Thank you.

**Comment by tungnguyendev** [comment]
Hi, 
Sorry because my mistake about code, actually my code as below:
```
    public class MainScreenCUITe : Screen
    {
        public WinComboBox comboBox1
        {
            get
            {
                return Find<WinComboBox>(By.ControlName("comboBox1"));
            }
        }

        public WinComboBox numericUpDown1
        {
            get
            {
                return Find<WinComboBox>(By.ControlName("numericUpDown1"));
            }
        }
    }
```
And using as bellow:
```
mainScreenCUITe.comboBox1.Click();
mainScreenCUITe.numericUpDown1.Click();
```
And my controls without AccessibleName.

If I create controls with AccessibleName after that I find control with name as bellow, it is ok:
public class MainScreenCUITe : Screen
```
    {
        public WinComboBox comboBox1
        {
            get
            {
                return Find<WinComboBox>(By.Name("comboBox1"));
            }
        }

        public WinComboBox numericUpDown1
        {
            get
            {
                return Find<WinComboBox>(By.Name("numericUpDown1"));
            }
        }
    }
```
And 
```
mainScreenCUITe.comboBox1.Click();
mainScreenCUITe.numericUpDown1.Click();
```

I'm using CUITe version 2.0.381.0

**Comment by icnocop** [comment]
I think the problem is that the NumericUpDown control should be based on the WinEdit control as follows:
```
        public WinEdit numericUpDown1
        {
            get { return Find<WinEdit>(By.ControlName("numericUpDown1")); }
        }
```

You can also see that the controls in the project "Sut.WinForms.Controls" from the CUITe source code do not have their AccessibleName property set and the test project "Sut.WinForms.ControlsTest" is able to find the controls using the `By.ControlName` search configuration without issues.

**Comment by tungnguyendev** [comment]
Hi, 
Thanks for your support, but if I add a TextBox as bellow code, I still can't find the numericUpDown control, event click of numericUpDown1 occur with textbox :(, it occur with some controls has same ControlType like textbox and richtextbox, maskedtextbox...
```
public class MainScreenCUITe : Screen
    {
        public WinEdit textBox1
        {
            get
            {
                return Find<WinEdit>(By.ControlName("textBox1"));
            }
        }

        public WinComboBox comboBox1
        {
            get
            {
                return Find<WinComboBox>(By.ControlName("comboBox1"));
            }
        }

        public WinEdit numericUpDown1
        {
            get
            {
                return Find<WinEdit>(By.ControlName("numericUpDown1"));
            }
        }
    }
```
And
```
mainScreenCUITe.textBox1.Click();
mainScreenCUITe.comboBox1.Click();
mainScreenCUITe.numericUpDown1.Click();
```

**Comment by icnocop** [comment]
What is the exact error message and stack trace?
Can you upload a minimal, complete, and verifiable example code (ex. something that I can run)?

**Comment by tungnguyendev** [comment]
hi, 
Have no error, just alway find the first in all control with the same ControlType. Below is my sample code.
https://drive.google.com/file/d/0B9iIwe5Exi43X1hBTllVZlN2cFE/view?usp=sharing

**Comment by icnocop** [comment]
Thank you.

Yes, I can reproduce the issue.

**Comment by tungnguyendev** [comment]
thanks!

**Comment by tungnguyendev** [comment]
Hi, 
I see this bug is resolved, so can I get latest version to use?

Thanks!

**Comment by icnocop** [comment]
Yes, it is fixed in v2.0.391.
Thank you.
